src/main/java/mods/eln/Eln.java

```diff
@@ -5241,6 +5251,16 @@ void recipeTurbine() {
 				Character.valueOf('A'), "blockAluminum",
 				Character.valueOf('M'), findItemStack("Advanced Machine Block")
 		);
+		addRecipe(findItemStack("Gas Turbine"),
+				"msH",
+				"sSs",
+				" M ",
+				Character.valueOf('m'), findItemStack("Advanced Electrical Motor"),
+				Character.valueOf('H'), findItemStack("Copper Thermal Cable"),
+				Character.valueOf('s'), "ingotSteel",
```

Comment:
One thing I've done previously is to use other-mod ingots if they exist, and vanilla/EA otherwise. I'll do that here as well.

It's all temporary until I get the chemical simulation running, anyway. Although I think I want to avoid writing the minecraft interface for that until we're on 1.10.